src/Microsoft.TemplateEngine.Edge/Settings/SettingsLoader.cs

```diff
@@ -212,8 +194,30 @@ public ITemplate LoadTemplate(ITemplateInfo info)
 
         public IFile HostTemplateConfigFile(IFileSystemInfo config)
```

Comment:
Just noticed this (even though it's not part of this change) - method names should indicate actions